frontend/src/App.jsx

```diff
@@ -322,7 +322,7 @@ class App extends PureComponent {
 
     this.setState({
       reportId: newReportProto.id,
-      commandLine: newReportProto.commandLine.join(" "),
+      commandLine: newReportProto.args.join(" "),
```

Comment:
hacky.

Reply:
We chatted about this offline, but just as a reminder: please remove the feature where users can modify arguments via the UI.